config/deploy.rb

```diff
@@ -43,6 +43,8 @@
 
 set :log_level, :debug
 
+set :sidekiq_config, -> { File.join(fetch(:deploy_to), 'config', 'sidekiq.yml') }
```

Comment:
__[RuboCop]__

Style/StringLiterals: Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.


<!-- Comment by [SideCI](https://sideci.com) -->